### PR TITLE
WIP: FCCAnalysis chain

### PR DESCRIPTION
**Pull request by clementhelsens**

### CODE REVIEW

File: FCCsnowmass.md [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+For this tutorial we will consider the following **physics processes**:
+
+-   e+ e- -> ZH -> Z to mumu and H to anything
+-   e+ e- -> ZZ -> Z to anything
+-   e+ e- -> WW -> W to anything
+
+
+Let's start by writing the pythia cards for the various processes.
+
+- **Pythia_ee_ZH_Zmumu_ecm240.cmd** for the Higgs Stralhung signal
```
Comment (vvolkl) [inline FCCsnowmass.md:51]:
```suggestion
- **Pythia_ee_ZH_Zmumu_ecm240.cmd** for the Higgs Strahlung signal
```

**Comment by vvolkl** [comment]
For the page to show up on the website, you need to add the filename on the top level Readme: https://github.com/HEP-FCC/fcc-tutorials/blob/master/README.md

File: FCCsnowmass.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+This tutorial will teach you how to:
+
+-   **generate** signal and background samples with **Pythia8** within FCCSW
+-   run a fast parametric **detector simulation** with **Delphes** within FCCSW
+-   apply an **event selection** on those samples with **FCCAnalyses**
+-   produce **flat ntuples** with observables of interest with **FCCAnalyses**
+-   produce plots with **FCCAnalyses**
+-   compare distributions produced with different generators
+
+{% endobjectives %}
```
Comment (vvolkl) [inline FCCsnowmass.md:14]: I think sphinx needs  an empty line after endobjectives.

**Review by vvolkl — CHANGES_REQUESTED** [review]

File: FCCsnowmass.md (hunk 1 above)
Comment (clementhelsens) [reply FCCsnowmass.md:54]: thanks

File: FCCsnowmass.md (hunk 2 above)
Comment (clementhelsens) [reply FCCsnowmass.md:14]: ok

**Pull request merged**